plugins/report-portal/src/components/LaunchesPage/LaunchesPageContent/LaunchesPageContent.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from 'react';
+
+import { Table, TableColumn } from '@backstage/core-components';
+import { useApi } from '@backstage/core-plugin-api';
+
+import { IconButton, Link } from '@material-ui/core';
+import Launch from '@material-ui/icons/Launch';
+import moment from 'moment';
```

Comment:
https://momentjs.com/docs/#/-project-status/

try considering other libraries like dayjs

Reply:
Moment is deprecated and no longer maintained, and they recommend using other libraries. As @riginoommen mentioned, `dayjs` is a good alternative, or consider using `luxon` for advanced features.